src/Mado/QueryBundle/Repositories/BaseRepository.php

```diff
@@ -124,16 +117,16 @@ public function setQueryOptionsFromRequest(Request $request = null)
         }
 
         $requestProperties = [
```

Comment:
You introduced a class that build an array, ... and here you create same array from that array.